node/batcher/batcher.go

```diff
@@ -158,15 +158,15 @@ func (b *Batcher) Submit(ctx context.Context, req *protos.Request) (*protos.Subm
 	}
 
 	b.Metrics.routerTxsTotal.Add(1)
-	traceId := req.TraceId
-	req.TraceId = nil
-	rawReq, err := proto.Marshal(req)
+
+	// make sure batched requests have only bytes of Envelope, not Request.
```

Comment:
// Make sure batched requests have only bytes of Envelope, not Request.
// We do this in order to maintain compatibility with Fabric block structure.